index.test-d.ts

```diff
@@ -0,0 +1,5 @@
+import {expectType} from 'tsd-check';
+import arrayDiffer from '.';
+
+expectType<string[]>(arrayDiffer(['a', 'b'], ['b', 'a']));
```

Comment:
@sindresorhus I'm getting `Argument of type string[] is not assignable to parameter of type string | number.
  Type string[] is not assignable to type string.` error, I couldn't figure out where I made the mistake.
Your input would be helpful.